### PR TITLE
Fix minor typo I found

### PR DESCRIPTION
**Pull request by sp3ctum**

### CODE REVIEW

File: prodigy.el [hunk 1]
```diff
@@ -1274,8 +1274,10 @@ SIGNINT signal."
...
 
 (defun prodigy-strip-ctrl-m (output)
-  "Strip 
 line endings from OUTPUT."
-  (s-replace "
" "" output))
+  "Strip 
+ line endings from OUTPUT."
+  (s-replace "
+" "" output))
```
Comment (rejeep) [inline prodigy.el]: I really don't understand this diff. Did you intend to change this part?

Comment (sp3ctum) [inline prodigy.el]: This is weird, I did absolutely not want to change this.

Comment (rejeep) [inline prodigy.el]: The diff is really weird as well because the `s-replace` part is already there in the actual code. I don't get it...? 😄

Comment (sp3ctum) [inline prodigy.el]: Hey you're right. Maybe it's some github related weirdness :confused:

Comment (rejeep) [inline prodigy.el]: Maybe. Can you try to force push the commit or something like that to remove it?

Comment (sp3ctum) [inline prodigy.el]: I redid the change in Emacs, see https://github.com/rejeep/prodigy.el/pull/82

Comment (rejeep) [inline prodigy.el]: Merged! 👍

**Pull request closed without merging**